client.py

```diff
@@ -51,4 +51,22 @@ async def on_ready():
 async def on_member_join(member):
     role = get(member.guild.roles, name='UNVALIDATED')
     await member.add_roles(role)
-    
\ No newline at end of file
+
+
+@bot.event
+async def on_message(message):
```

Comment:
As instructed, we have a services folder to put all the commands offered by our bot. eg. services/membership.py.

- [ ] Create a Poll Service/Cog in our services and migrate all your commands there